image/rhel/konflux.Dockerfile

```diff
@@ -45,6 +45,7 @@ ENV MAIN_TAG_SUFFIX="$VERSIONS_SUFFIX" COLLECTOR_TAG_SUFFIX="$VERSIONS_SUFFIX" S
 
```

Comment:
(not about this line)
1. Operator and scanner-v4 are missing.
2. The same change would be needed in stackrox/scanner repo.
3. Is this targeted for 4.6.0?
4. It would be really handy to reference a JIRA ticket for this change. There is at least an epic for this work, right?
5. I tend to forget things over time. It would be nice to provide a link to the documentation or some context about the environment variable in either the PR description or the JIRA ticket description so that future-us can restore knowledge from git history. I know [this presentation](https://docs.google.com/presentation/d/1o3IowxHX6BsnxGkIInaQ0lBgnn_K5Ex8jxwCYCeNsqs/edit#slide=id.g2679cb578c3_0_17) and I wonder if a better source exists.

Reply:
Hi Misha!

Apologies for all the mistakes, and thank you for your diligence (as always). I originally made this a draft to check the results and too hastily marked this as ready for review. I'll take care of 1, 2, and 4.

> 3. Is this targeted for 4.6.0?
I agree with David; it's not required in 4.6 (but I wouldn't mind it getting in either).

> 5. I tend to forget things over time. It would be nice to provide a link to the documentation or some context about the environment variable in either the PR description or the JIRA ticket description so that future-us can restore knowledge from git history. I know [this presentation](https://docs.google.com/presentation/d/1o3IowxHX6BsnxGkIInaQ0lBgnn_K5Ex8jxwCYCeNsqs/edit#slide=id.g2679cb578c3_0_17) and I wonder if a better source exists.

I wasn't sure what the protocol linking internal docs and such was, but I guess it isn't much different than linking Slack messages. I'll edit the description of the PR to include the following info: more about the `strictfipsruntime` can be found in [this doc](https://docs.google.com/document/d/1CTpSwITQfOgoOTlPITZNJZy0ltYz60hxkmWsLemCqw0/edit?tab=t.0) (there might be a better resource but this is the one David and I found), and for our general research on the topic, see [this doc](https://docs.google.com/document/d/1KcjlevcMF4DLi-KROru7tuUUIb4wSk5e6bnXpU8HZxA/edit?tab=t.0).

> 6. Should we see from go compiler logs that the setting was recognized during the builds?

I'm 100% not sure. I don't think so, but I'll verify (by coordinating with David to check-payload tool against the Konflux image) and post that information in the description.

Reply:
The doc you linked is indeed a good one but at the same time it raised more questions than answers for me. https://docs.google.com/document/d/1CTpSwITQfOgoOTlPITZNJZy0ltYz60hxkmWsLemCqw0/edit?disco=AAABXyFMlGE

Reply:
Re 4.6: if you intend it to go in 4.6.z (where z>=0), please add `backport-for-4.6-konflux-release` label to the PR.